pkg/skaffold/runner/runcontext/context.go

```diff
@@ -208,10 +209,14 @@ func GetRunContext(opts config.SkaffoldOptions, configs []schemaUtil.VersionedCo
 	kubeContext := kubeConfig.CurrentContext
 	logrus.Infof("Using kubectl context: %s", kubeContext)
 
-	// TODO(dgageot): this should be the folder containing skaffold.yaml. Should also be moved elsewhere.
-	cwd, err := os.Getwd()
-	if err != nil {
-		return nil, fmt.Errorf("finding current directory: %w", err)
+	var cwd string
+	if opts.ConfigurationFile == "" {
+		cwd, err = os.Getwd()
+		if err != nil {
+			return nil, fmt.Errorf("finding current directory: %w", err)
+		}
+	} else {
+		cwd = filepath.Dir(opts.ConfigurationFile)
```

Comment:
I tried to get this change through once before but couldn't because the Cloudcode IDEs have made this assumption in several places.

Reply:
go/skaffold-run-from-here is the correct alternative.

Reply:
🤦‍♀️ 🤦‍♀️ 🤦‍♀️